local_pathfinding/ompl_path.py

```diff
@@ -40,9 +65,19 @@ def get_waypoints(self):
         return waypoints
 
     def update_objectives(self):
+        """Update the objectives on the basis of which the path is optimized.
+
+        Raises:
+            NotImplementedError: Method or function hasn't been implemented yet.
+        """
         raise NotImplementedError
 
     def _init_simple_setup(self) -> og.SimpleSetup:
+        """Initialize and configure the OMPL SimpleSetup object.
```

Comment:
Describe what the SimpleSetup object is. From the OMPL primer: "encapsulates the various objects necessary to solve a geometric or control query in OMPL".